index.html

```diff
@@ -87,7 +88,7 @@ <h3 class="article__title">Why do refugees need bikes?</h3>
                 </p>
               </div>
 
-              <div class="article">
+              <div class="article1">
```

Comment:
I can see `<div>` elements on the page with very similar (but not quite identical) classes, for example, `articleArea`, `article`, `articles`. Classes are used where we want to style multiple elements in a similar way. IDs are used when there is only one element that should be styled. Have a think about whether these elements with "article"-like classes should all have the same class or not